components/resources/brave_components_strings.grd

```diff
@@ -444,25 +443,43 @@
       </if>
 
       <!-- WebUI rewards resources -->
-      <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_TITLE" desc="">Brave Private Ads</message>
-      <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_DESC" desc="">
-        Brave Private Ads are unobtrusive and private by design. Ads are based on your interests, but no personal data or browsing history ever leaves your device. Best of all, Brave Private Ads let you earn BAT (<ph name="BEGIN_LINK">$1</ph>Basic Attention Token<ph name="END_LINK">$2</ph>) just for viewing.
+      <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_TITLE" desc="">Manage Brave Ads</message>
+      <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_DESCRIPTION" desc="">
+        Control what kinds of Brave Ads you see, and how often. The more you see, the more you can earn.
+      </message>
+      <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_DESCRIPTION_UNCONNECTED" desc="">
+        Control what kinds of Brave Ads you see, and how often.
+      </message>
```

Comment:
Would be nice if we could reuse the same sentence.

Reply:
Done.